### ISSUE TITLE
[BUG] Immersive Engineering's 3D models don't appear in the Engineer's manual

### ISSUE THREAD

**Issue by Callisto1506** [itsdinkd/AQM3#63]
**Modpack Version**
1.7

**Minecraft Version**
1.18.1 (forge-39.0.79

**Describe the bug**
The 3D models of the heavy machines's structures don't appear in the manual. By pressing the stop button you get the model of the complete machine, but the model with the steps for building it is not visible.

**To Reproduce**
1. Launch your game (solo or multiplayer, I tested both)
2. Get the engineer's manual and open it
3. Click on Heavy Machinery -> Metal press (or any other multibock machine in the book)
4. The step-by-step building model in not visible.

**Expected behavior**
A 3D model of the multiblock machine should appear in the book, showing how to build it layer by layer

**Screenshots**
![image](https://user-images.githubusercontent.com/101988533/159174072-742d4452-d1d0-4cd5-a058-352a596acbac.png)

**Crash log or latest.log.log**
Nothing here

**Additional context**
A few more things : this bug does come from the modpack, I tested launching the game with just the mod Immersive Engineering and the 3d models are there. Also, my friends on our server have the same bug.

**Comment by itsdinkd** [comment]
Please send me your latest.log after opening the book

**Comment by Callisto1506** [comment]
[latest.log](https://github.com/itsdinkd/AQM3/files/8355244/latest.log)

Here it is ! I opened the book at 12:08 pm, in the end of the log file. As you will see, there is no error mesage when I open the book.

Just to give you some context : we (my friends and I) slightly modified the modpack to make it more convenient for us to play. (If you want the details of the changes we made, I can tell you, but they are mainly changes in the kubejs recipes, nothing to do with Immersive Engineering). I believe some of the errors in the log file are linked with that.

But the bug was there with the modpack before we made changes (I reported this bug before that), and is still present after the changes. If you want I can provide you a .log file from before we made the changes.

I also tried to launch the version of Immersive Engineering in the modpack alone with the same version of forge, and the models appeared normally. So it does come from something in the modpack.

Sorry to respond 4 days later, I was very busy these last few days !  If you want more info, I will be pleased to help you.
